roles/openshift_health_checker/openshift_checks/fluentd_config.py

```diff
@@ -0,0 +1,157 @@
+"""
+Module for performing checks on a Fluentd logging deployment configuration
+"""
+
+import json
+
+from openshift_checks import OpenShiftCheck, get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+
+
+class FluentdConfig(OpenShiftCheck):
```

Comment:
Since individual nodes can have different logging config the idea was to check it on all the nodes; so yeah, it would need to override `is_active`. We could still reuse functionality from `LoggingCheck` in various was if it were more involved than this.